src/configure/resources/messages.ts

```diff
@@ -113,6 +113,7 @@ export class Messages {
     public static AdoDifferentTenantError = " One potential reason can be the AAD tenant of your Azure DevOps repository is different than the one you have been logged-in to VSCode. Open 'Command Pallete' and select 'Azure: Sign Out' and then Sign-In to other tenant by invoking 'Deploy to Azure' extension";
     public static ConfiguringPipelineFailed = "Configuring provisioning pipeline failed due to %s";
     public static CreatingSPN = "Creating SPN";
-    public static ConfiguringDraftPipeline = "Configuring draft pipeline";
+    public static GeneratingWorkflowFiles = "Generating workflow files";
     public static CreatingResourceGroup = "Creating resource group";
+    public static ConfiguringWorkflowAndDeployment = 'Configuring workflow and proceeding to deployment...';
```

Comment:
It should be Configuring github workflow

Reply:
done